ai-predictor-demo/static/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>AI Predictor</title>
+</head>
+<body>
+    <div style="max-width: 400px; margin: 0 auto; padding: 20px; text-align: center;">
+        <h1 style="margin-bottom: 20px;">AI Predictor</h1>
+        <div style="margin-bottom: 20px;">
+            <label for="input">Input string:</label><br>
+            <input type="text" id="input" style="width: 100%; padding: 10px; border-radius: 5px;">
+        </div>
+        <div style="margin-bottom: 20px;">
+            <label for="labels">Labels (comma separated list):</label><br>
+            <input type="text" id="labels" style="width: 100%; padding: 10px; border-radius: 5px;">
+        </div>
+        <div style="margin-bottom: 20px;">
+            <label for="multilabel">Multi Label:</label><br>
+            <select id="multilabel">
+                <option value="true">True</option>
+                <option value="false">False</option>
+            </select>
+        </div>
+        <button onclick="predict()" style="background-color: #1d6020; color: white; padding: 14px 20px;">Predict</button>
+        <div id="results" style="margin-top: 20px; text-align: left;"></div>
+    </div>
+    <script>
+        function predict() {
+            var input = document.getElementById("input").value;
+            var labels = document.getElementById("labels").value.split(",").map(label => label.trim());
+            var multiLabel = document.getElementById("multilabel").value === "true";
+
+            var data = {
+                "input": input,
+                "labels": labels,
+                "multi_label": multiLabel
+            };
+
+            fetch("http://localhost:5000/predict", {
```

Comment:
Just as a clarification question: Is there a particular reason why you chose not to directly invoke the API here instead of using the flask app BE as a layer of indirection?

Reply:
Ah -- in that case, can we modify to confirm that things work fine when using the API directly from FE? I think we may have missed including `OPTIONS` in the allowed methods (TBH, we probably could just make it `*` for this prototype). If that's the case, you'll have to rebuild the container after making the change, push the image, and then cycle the pod so it fetches the new image by either:

* Destroying the current one using `kubectl`
* Do a `helm uninstall` and then redeploy

LMK and we can pair on the deployment stuff tomorrow.